src/decorator/columns/Column.ts

```diff
@@ -128,6 +141,18 @@ export function Column(typeOrOptions?: ((type?: any) => Function)|ColumnType|(Co
         if (options.type === "hstore" && !options.hstoreType)
             options.hstoreType = reflectMetadataType === Object ? "object" : "string";
 
+        if (options.type === "json" && !options.jsonType) 
```

Comment:
I think better to remove this block for now to ensure its 100% non breaking change.

Reply:
Spent quite a while thinking about this at the time. If a user manages map a string property of some entity to a json column in the database, in the expectation that json column will only ever contain strings, then of course their code will break if anyone ever inserts anything into that json column which is not a simple json string. The actual 'break' this change would imply is that in this corner case we would see quotes around their string when we didn't before (because the json parser is not being invoked). I felt that the balance was in favor of protecting the user from the consequence of their bad code design.

Reply:
Happy to pull this out (and any associated unit test) if it is really the only way to get this merged, but since (for me) the only code that would break is arguably already broken, and this change should actually improve the behavior of that broken code, I think there's an argument for keeping this in. I'm going to fix everything else, rebase, and request a re-review.